src/get_functions.jl

```diff
@@ -0,0 +1,40 @@
+"""
+    get_origin( vtk_file )
+
+Retrieve the vector of coordinates of the origin of a regular structured grid from a given VTK XML file.
+
+See also: [`VTKFile`](@ref)
+"""
+function get_origin( vtk_file )
+
+  # open the file and locate the ImageData section
+  root = LightXML.root( vtk_file.xml_file )
+  piece = root["ImageData"][1]
```

Comment:
```suggestion
  dataset_element = root["ImageData"][1]
```
As above. Also applies to `get_spacing` below.